.templatesyncignore

```diff
@@ -1,17 +1,5 @@
-# THIS FILE CANNOT BE SYNCED
-# use glob patterns as in .gitignore
-# recommend listing non-basics tutorial directories here
-###
-# configuration files
 cookiecutter.yaml
-###
-# environment lock files
 conda/*lock.yml
-###
-# team member files
 team/*-*
-###
-# book files (configuration, usage-specific)
 book/_config.yml
-###
-# event-specific tutorials
+book/logo.png
```

Comment:
To prevent logo from being changed. I'm thinking this might be a good one to put directly in the template.